<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>683</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mainwindow.qrc">
    <normaloff>:/MainWindow/Resources/icon.jpg</normaloff>:/MainWindow/Resources/icon.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="leftlabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Input picture here</string>
        </property>
        <property name="locale">
         <locale language="Chinese" country="China"/>
        </property>
        <property name="text">
         <string>Input picture here</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="rightlabel">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>300</height>
         </size>
        </property>
        <property name="statusTip">
         <string>Input picture or video here</string>
        </property>
        <property name="locale">
         <locale language="Chinese" country="China"/>
        </property>
        <property name="text">
         <string>Input picture or video here</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="minimumSize">
       <size>
        <width>800</width>
        <height>250</height>
       </size>
      </property>
      <property name="statusTip">
       <string>Output</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>798</width>
         <height>248</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="statusTip">
         <string>Clear output</string>
        </property>
        <property name="text">
         <string>&amp;Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="statusTip">
         <string>Start</string>
        </property>
        <property name="text">
         <string>&amp;Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_Open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionMatched"/>
     <addaction name="actionMatching"/>
    </widget>
    <addaction name="menu_Open"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQT"/>
   </widget>
   <widget class="QMenu" name="menu_Setting">
    <property name="title">
     <string>&amp;Setting</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Setting"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/save.png</normaloff>:/MainWindow/Resources/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/exit.png</normaloff>:/MainWindow/Resources/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="statusTip">
    <string>Quit the application</string>
   </property>
  </action>
  <action name="actionAboutQT">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/Qt.png</normaloff>:/MainWindow/Resources/Qt.png</iconset>
   </property>
   <property name="text">
    <string>About QT</string>
   </property>
   <property name="statusTip">
    <string>Show the Qt library's About box</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/help.png</normaloff>:/MainWindow/Resources/help.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionMatched">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/open.png</normaloff>:/MainWindow/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Matched</string>
   </property>
  </action>
  <action name="actionMatching">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/MainWindow/Resources/open.png</normaloff>:/MainWindow/Resources/open.png</iconset>
   </property>
   <property name="text">
    <string>Matching</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
